### DISCUSSION TITLE
How do I get Galaxy brain badge?

### DISCUSSION THREAD

**Discussion by CondratKenasil** [Schweinepriester/github-profile-achievements#505] [Q&A] [answered]
?

**Comment by MitkoVtori** [comment, marked as answer]
by commenting a question(like I do now) and the Author of the question mark your coment as an answer. Right here /
........................................................................................................................................................................................................................................../
....................................................................................................................................................................................................................................../
.................................................................................................................................................................................................................................../
................................................................................................................................................................................................................................/
............................................................................................................................................................................................................................../
...............................................................................................................................................................________________________________
.........................................................................................................................__________________
..............................................................___________________________
......................................................../
.................................................../
................................................/
............................................../
............................................./
.........................................../
......................................./
................................./
............................/
........................./
..................../
.............../
........./
..../
/V

**Reply by CondratKenasil** [reply]
Thank you!

**Comment by Preetiraj3697** [comment]
@CondratKenasil  Mark your answer in other discussion.
